sw/airborne/firmwares/rotorcraft/stabilization/stabilization_attitude_ref_quat_float.h

```diff
@@ -59,7 +59,7 @@ struct AttRefQuatFloat {
 
 
 extern void attitude_ref_quat_float_init(struct AttRefQuatFloat *ref);
-extern void attitude_ref_quat_float_enter(struct AttRefQuatFloat *ref, float psi);
+extern void attitude_ref_quat_float_enter(struct AttRefQuatFloat *ref, struct FloatEulers *state_euler);
```

Comment:
Why passing an euler format here, rather than a quat ?
With the current implementation, it means that you'll still have to make the conversion at some point, but if we replace it by a ref really based on quaternions, this would make a transformation that is not needed.

Reply:
Ah I figured that since it is there in the struct, it would be best to compute both, but it is also already computed with every update. Does it still make sense to keep the euler angles in the reference struct though?